server.go

```diff
@@ -2842,7 +2853,8 @@ func (s *server) peerConnected(conn net.Conn, connReq *connmgr.ConnReq,
 		ReadPool:                s.readPool,
 		Switch:                  s.htlcSwitch,
 		InterceptSwitch:         s.interceptableSwitch,
-		ChannelDB:               s.chanDB,
+		ChannelDB:               s.localChanDB,
+		ChannelGraph:            s.remoteChanDB.ChannelGraph(),
```

Comment:
is this field necessary? it wasn't set before

Reply:
The `ChannelGraph` field? It was added in this diff to help isolte the local vs remote DB.